.github/ISSUE_TEMPLATE/bugreport.yml

```
@@ -0,0 +1,76 @@
+name: Bug Report
+description: You found a bug on Scuffle? This way please
+title: "[Bug]: Scuffle has a bug here [...]"
+labels: ["bug"]
+assignees:
+  - ScuffleTV/reviewers
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### **Thank you for taking the time to fill out this Bug Report!**
+        Please provide as much information as possible. This will help us resolve the Bug quickly and accurately.
+  - type: markdown
+    attributes:
+      value: ---
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What seems to be the problem?
+      description: | 
+                  A clear and concise description of what needs to be fixed.
+                  *Make sure there are no misunderstandings here. It would be best if you have a video/screenshot for us.*
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operation System
+      description: What is the affected environment?
+      options:
+        - "Microsoft Windows"
+        - "Apple macOS"
+        - "Linux"
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser
+      description: On which of the browser do you see this problem?
+      options:
+        - "Google Chrome"
```

No need to have Vivaldi in the list.

---

Is it a chromium browser?

---

Yeah

---

Vivaldi supports Chrome Addons

---

The reason why brave should be left in, is because it has a lot of privacy tweaks and a built-in working ad-blocker, which makes it unique. The Vivaldi ad blocker is opt-in, and the browser is more features over privacy oriented, standard chromium wrapper

---

could just say Chromium (Google Chrome, Brave, Vivaldi, etc.)
```suggestion
        - "Chromium (Google Chrome, Microsoft Edge, Brave, Vivaldi, etc.)"
```

---

everyone agree with this?

---

Think it's fine as long as you keep opera as a separate thing

---

seems fine

---

Ah shit don't forget safari

---

Is 7TV compatible with Safari?

---

added

---

Further changes desired?